docs/api/api_blueprint.apib

```diff
@@ -19326,3 +19326,87 @@ Creates the copy. Requires Co-Organizer access
             "self": "/v1/events/1/copy"
           }
         }
+
+# Group Change Password
+
+This Groups APIs are used for change password request of the user.
+
+## Change Password [/v1/change-password]
+
+### Change password using old password [POST]
+
++ Request
+
+    + Headers
+
+            Content-Type: application/json
+            Authorization: JWT <Auth Key>
+
+    + Body
+
+            {
+                "data": {
+                    "old-password": "Old Password",
+                    "new-password": "New Password"
+                }
+            }
+
++ Response 200 (application/json)
+
+       {
+          "email": "example@example.com",
+          "id": 1,
+          "name": "example",
+          "password_changed": true
+       }
+
+## Reset Forgotten Password [/v1/reset-password]
+Change password through token received via email.
+
+### Get Email Token [POST]
+
++ Request
+
+    + Headers
+
+            Content-Type: application/json
+
+    + Body
+
+            {
+                "data": {
+                    "email": "example@example.com"
+                }
+            }
+
++ Response 200 (application/json)
+
+        {
+          "message": "Email Sent"
+        }
+
+
+### Change Password from Token [PATCH]
```

Comment:
This is reset password rather than change password. Please change the name else it is confusing.